modules/edit-modes/src/lib/scale-mode.js

```diff
@@ -1,101 +1,223 @@
 // @flow
 
+import turfBuffer from '@turf/buffer';
+import bbox from '@turf/bbox';
 import turfCentroid from '@turf/centroid';
+import turfBearing from '@turf/bearing';
+import bboxPolygon from '@turf/bbox-polygon';
+import { point, featureCollection } from '@turf/helpers';
+import { coordEach } from '@turf/meta';
 import turfDistance from '@turf/distance';
 import turfTransformScale from '@turf/transform-scale';
+import { getCoord, getGeom } from '@turf/invariant';
 import type { FeatureCollection, Position } from '../geojson-types.js';
 import type {
   ModeProps,
   PointerMoveEvent,
   StartDraggingEvent,
   StopDraggingEvent,
-  DraggingEvent
+  DraggingEvent,
+  EditHandleFeature
 } from '../types.js';
-import { BaseGeoJsonEditMode, type GeoJsonEditAction } from './geojson-edit-mode.js';
+import { getPickedEditHandle } from '../utils';
+import { BaseGeoJsonEditMode } from './geojson-edit-mode.js';
 import { ImmutableFeatureCollection } from './immutable-feature-collection.js';
 
 export class ScaleMode extends BaseGeoJsonEditMode {
```

Comment:
Can you also add tests for scale mode? We had tests before at `modules/layers/test/lib/mode-handlers/scale-handler.test.js` that need ported over, but they probably need mostly rewritten since this mode works differently now.